pkg/monitor/intervalcreation/node.go

```diff
@@ -520,6 +521,61 @@ func anonymousCertConnectionError(nodeLocator, logLine string) monitorapi.Interv
 	}
 }
 
+// lower 'f'ailed and 'error'
+var failedLeaseUpdateErrorRegex = regexp.MustCompile(`failed to update lease, error: Put \"(?P<URL>[a-z0-9.-:\/\-\?\=]+)\": (?P<MSG>[^\"]+)`)
+
+// upper 'F'ailed and 'err'
+var failedLeaseUpdateErrRegex = regexp.MustCompile(`Failed to update lease\" err\=\"Put \\\"(?P<URL>[a-z0-9.-:\/\-\?\=]+)\\\": (?P<MSG>[^\"]+)`)
```

Comment:
Why not just [Ff]ailed ?

Reply:
Ah I see they're different in other places

Reply:
The err vs. error string is a bit different as well but if you see a common regex for them both that would work for me as well.

Reply:
No it's fine, 2 simpler regex is better than 1 unreadable one.